unit_tests/ut_h_edit_dist_features.amlg

```diff
@@ -285,15 +285,17 @@
 
 	(assign (assoc result (call_entity "howso" "get_internal_parameters") ))
 
-
-	(call_entity "howso" "react_into_trainee" (assoc
-		residuals (true)
-		context_features features
-	))
-
 	(assign (assoc
-		result (call_entity "howso" "get_prediction_stats" (assoc "stats" (list "r2" "rmse" "mae") ))
+		result
```

Comment:
please check that 'result' is indented once relative to the '(assign' - this may just be a github display issue though